daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -428,9 +428,12 @@ private[lf] class Runner(
       mat: Materializer,
   ): (Speedy.Machine, Future[SValue]) = {
     val machine =
-      Speedy.Machine.fromPureSExpr(extendedCompiledPackages, script.expr, traceLog, warningLog)(
-        Script.DummyLoggingContext
-      )
+      Speedy.Machine.fromPureSExpr(
+        extendedCompiledPackages,
+        script.expr,
+        traceLog = traceLog,
+        warningLog = warningLog,
+      )(Script.DummyLoggingContext)
```

Comment:
As disclosed contracts is (potentially) exposed here, we now need default parameters to be named.